### PR TITLE
Business info

### PR DESCRIPTION
**Pull request by marwajomaa**

### CODE REVIEW

File: client/src/Components/Pages/SEInformation/BusinessInfo/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import './style.css';
+import signal from './Imgs/signal.png';
```
Comment (thesoorae) [inline client/src/Components/Pages/SEInformation/BusinessInfo/index.js]: usually better not to capitalize files or directory names.

[the remark this answers is not in the document]
Comment (sultanassi95) [reply client/src/Components/Pages/SEInformation/BusinessInfo/index.js]: @m4v15 Hey Matt, I think we have been giving different views on this matter, if you would like to have your's said out loud check what Soo-Rae is telling.

File: client/src/Components/Pages/SEInformation/BusinessInfo/index.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                  <span className="hidden" />
+                </label>
+              </div>
+
+              <div className="form__field">
+                <textarea id="DescriptionOfTheCompany " type="text" name="DescriptionOfTheCompany" className="form__input text__area" placeholder="Description of the company" required />
+              </div>
+
+              <div className="form__field SaveBtn">
+                <input type="submit" value="Save & Continue" />
```
[the remark this answers is not in the document]
Comment (marwajomaa) [reply client/src/Components/Pages/SEInformation/BusinessInfo/index.js]: All pages components will be in one form and all data will be collected in the same state
@thesoorae

File: client/src/Components/Pages/SEInformation/BusinessInfo/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (m4v15) [reply client/src/Components/Pages/SEInformation/BusinessInfo/index.js]: @sultanassi95 hey, sorry maybe I was a bit confusing - when the main file in a folder is a react component, the folder should be PascalCase - however in this case the folder just contains images, so the folder should be lowercase:
`
/src 
 /components 
    /Header
    /Footer
    /Notifications
      /index.js
      /assets
          img.png
`

**Pull request merged**